packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from 'react';
+
+import { Dialog } from '@/components/Dialog';
+import { action } from '@storybook/addon-actions';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  PayableDetailsForm,
+  PayableDetailsFormProps,
+} from './PayableDetailsForm';
+
+const meta: Meta<typeof PayableDetailsForm> = {
+  title: 'Payables/Payables — Details Form',
+  component: PayableDetailsForm,
+};
+
+const payableId = '9a3b97a5-a1ba-4d8c-bade-ad3c47ae61e0';
+
+type Story = StoryObj<typeof PayableDetailsForm>;
+
+const actions = {
+  onSavePayable: action('onSavePayable'),
+  onCreatePayable: action('onCreatePayable'),
+};
+
+const WithDialogComponent: FC<PayableDetailsFormProps> = (args) => {
```

Comment:
I see no point in making the component as an external, since it is a part of the story.

Reply:
Could you explain what’s missing? I don’t quite understand what changes I need to make

Reply:
Okey, I will do this, but modular vs in scoped components is subjective topic